docs/setup-nginx-proxy-with-minio.md

```diff
@@ -62,20 +72,20 @@ If you want to serve web-application and Minio from the same nginx port then you
  }
 ```
 
-## 4. Recipe Steps
+## <a name="runnginx"></a>4. Run `NGINX`
 
-### Step 1: Start Minio server.
+### Start Minio Server
 
 ```sh
 minio server /mydatadir
 ```
 
-### Step 2: Restart Nginx server.
+### Restart the NGINX Server
 
 ```sh
 sudo service nginx restart
```

Comment:
Should probably change to systemd syntax: systemctl restart nginx.service

Reply:
@deekoder , @eco-minio : this is resolved in the latest commit. Please see https://github.com/jamesneimanconsulting/cookbook/blob/patch-6/docs/setup-nginx-proxy-with-minio.md.